### PR TITLE
Tag DASKR.jl v0.3.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaDiffEq/DASKR.jl](https://github.com/JuliaDiffEq/DASKR.jl)
Release: [v0.3.0](https://github.com/JuliaDiffEq/DASKR.jl/releases/tag/v0.3.0)
Travis: [Travis Build Status](https://travis-ci.org/JuliaDiffEq/DASKR.jl/builds/232681388)
Diff: [vs v0.2.0](https://github.com/JuliaDiffEq/DASKR.jl/compare/396480ab30c4290feeb38ef2d942fd4beb131aec...af2c76d3068e98d213ef656b78698e35da69412c)
`requires` vs v0.2.0: 
```diff
--- 0.2.0/requires
+++ 0.3.0/requires
@@ -1,3 +1,3 @@
 julia 0.5
-DiffEqBase 0.15.0
-Compat 0.17.0
+DiffEqBase 1.5.1
+Compat 0.19.0
```
cc: @ChrisRackauckas

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: DASKR/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+DiffEqBase 1.5.1
+Compat 0.19.0
```
Comment (tkelman) [inline DASKR/versions/0.3.0/requires]: no indexstyle, why was this changed?

Comment (ChrisRackauckas) [inline DASKR/versions/0.3.0/requires]: It was missed in review. Proper scoldings occurred. Thanks.
